python/ray/experimental/dag/actor_node.py

```diff
@@ -0,0 +1,117 @@
+import ray
+from ray.experimental.dag.dag_node import DAGNode
+
+from typing import Any, Dict, List, Optional, Tuple
+
+
+class ActorNode(DAGNode):
+    """Represents an actor creation in a Ray task DAG."""
+
+    def __init__(self, actor_cls: type, cls_args, cls_kwargs, cls_options):
+        self._actor_cls = actor_cls
+        self._last_call: Optional["ActorMethodNode"] = None
+        DAGNode.__init__(self, cls_args, cls_kwargs, cls_options)
+
+    def _copy_impl(
+        self,
+        new_args: List[Any],
+        new_kwargs: Dict[str, Any],
+        new_options: Dict[str, Any],
+    ):
+        return ActorNode(self._actor_cls, new_args, new_kwargs, new_options)
+
+    def _execute_impl(self):
```

Comment:
Is this the default implementation if not passed to a different executor?

Reply:
docstring please

Reply:
yep each subclass of DAGNode implements its own executor by calling ray.remote()